include/cinder/Log.h

```diff
@@ -82,9 +82,6 @@ class Logger {
 
 	virtual void write( const Metadata &meta, const std::string &text ) = 0;
 
-	template<typename LoggerT, typename... Args>
-	static std::shared_ptr<LoggerT> makeLogger( Args&&... args );
```

Comment:
My thinking behind this was: provide the user with a way to make a logger before inserting it into the LogManager.  Perhaps they wanted to adjust certain things before adding it to the LogManager?